test_dhcp_relay does not support PortChannel in Vlan

**Description**

<!--
Briefly describe the problem you are having in a few paragraphs.
-->

**Steps to reproduce the issue:**
Command line to start test:
```
$ ./run_tests.sh -i ../ansible/str,../ansible/veos -n vms2-2-t0-2700 -e "--skip_sanity" -l info -u -c dhcp_relay/test_dhcp_relay.py -u -e "--skip_sanity"
```

**Describe the results you received:**
I see the error in test output
```
_______________________________________________________________________________ ERROR at setup of test_dhcp_relay_default ________________________________________________________________________________

duthosts = <tests.common.devices.duthosts.DutHosts object at 0x7fc8bc7a8fd0>, rand_one_dut_hostname = 'str-msn2700-20', ptfhost = <tests.common.devices.ptf.PTFHost object at 0x7fc8b9d87610>
tbinfo = {'comment': '####', 'conf-name': 'vms2-2-t0-2700', 'duts': ['str-msn2700-20'], 'duts_map': {'str-msn2700-20': 0}, ...}

    @pytest.fixture(scope="module")
    def dut_dhcp_relay_data(duthosts, rand_one_dut_hostname, ptfhost, tbinfo):
        """ Fixture which returns a list of dictionaries where each dictionary contains
            data necessary to test one instance of a DHCP relay agent running on the DuT.
            This fixture is scoped to the module, as the data it gathers can be used by
            all tests in this module. It does not need to be run before each test.
        """
        duthost = duthosts[rand_one_dut_hostname]
        dhcp_relay_data_list = []

        mg_facts = duthost.get_extended_minigraph_facts(tbinfo)
        host_facts = duthost.setup()['ansible_facts']

        # SONiC spawns one DHCP relay agent per VLAN interface configured on the DUT
        vlan_dict = mg_facts['minigraph_vlans']
        for vlan_iface_name, vlan_info_dict in vlan_dict.items():
            # Gather information about the downlink VLAN interface this relay agent is listening on
            downlink_vlan_iface = {}
            downlink_vlan_iface['name'] = vlan_iface_name

            for vlan_interface_info_dict in mg_facts['minigraph_vlan_interfaces']:
                if vlan_interface_info_dict['attachto'] == vlan_iface_name:
                    downlink_vlan_iface['addr'] = vlan_interface_info_dict['addr']
                    downlink_vlan_iface['mask'] = vlan_interface_info_dict['mask']
                    break

            # Obtain MAC address of the VLAN interface
            res = duthost.shell('cat /sys/class/net/{}/address'.format(vlan_iface_name))
            downlink_vlan_iface['mac'] = res['stdout']

            downlink_vlan_iface['dhcp_server_addrs'] = mg_facts['dhcp_servers']

            # We choose the physical interface where our DHCP client resides to be index of first interface in the VLAN
            client_iface = {}
            client_iface['name'] = vlan_info_dict['members'][0]
>           client_iface['alias'] = mg_facts['minigraph_port_name_to_alias_map'][client_iface['name']]
E           KeyError: u'PortChannel101'

```

**Describe the results you expected:**


**Additional information you deem important:**
    **Output of `show version`:**

```
20191130.62
```

--- a/tests/dhcp_relay/test_dhcp_relay.py
+++ b/tests/dhcp_relay/test_dhcp_relay.py
@@ -186,7 +186,7 @@ def testing_config(request, duthosts, rand_one_dut_hostname, tbinfo):
             yield testing_mode, duthost, 'dual_testbed'
     elif tbinfo['topo']['name'] == 't0-56-po2vlan':
         if testing_mode == SINGLE_TOR_MODE:
-            if subtype_exist:
+            if subtype_exist and subtype_value == 'DualToR':
                 assert False, "Wrong DHCP setup on t0-56-vlan2po testbeds"
 
             yield testing_mode, duthost, 'single_testbed'

